The addition of gstvideodmabufpool.h, which depends on gstvideopool.h,
causes this include to pull things in the wrong order.

https://gitlab.freedesktop.org/gstreamer/gstreamer/-/merge_requests/10411

diff -up ./modules/codec/gstreamer/gstvlcvideopool.h.orig ./modules/codec/gstreamer/gstvlcvideopool.h
--- ./modules/codec/gstreamer/gstvlcvideopool.h.orig	2025-12-23 05:49:20.000000000 -0500
+++ ./modules/codec/gstreamer/gstvlcvideopool.h	2026-02-10 01:45:40.557003527 -0500
@@ -28,7 +28,7 @@
 #define VLC_GST_VIDEO_POOL_H
 
 #include <gst/gstbufferpool.h>
-#include <gst/video/gstvideopool.h>
+#include <gst/video/video.h>
 
 #include "gstvlcpictureplaneallocator.h"
 
